src/libs/actions/Travel.ts

```diff
@@ -15,15 +15,27 @@ function acceptSpotnanaTerms() {
             key: ONYXKEYS.NVP_TRAVEL_SETTINGS,
             value: {
                 hasAcceptedTerms: true,
+                isLoading: true,
             },
         },
     ];
+
+    const finallyData: OnyxUpdate[] = [
+        {
+            onyxMethod: 'set',
```

Comment:
Since this is `set` instead of `merge` isn't this going to remove `hasAcceptedTerms` from the key?

Reply:
oops! nice catch